packages/components/src/components/animations/LottieAnimation.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components';
+import React, { useEffect, useState } from 'react';
+import Lottie, { LottieOptions } from 'lottie-react';
+
+import { DeviceModelInternal } from '@trezor/connect';
+import { AnimationWrapper, Shape } from './AnimationPrimitives';
+import { resolveStaticPath } from '../../utils/resolveStaticPath';
+
+const StyledLottie = styled(Lottie)`
+    width: 100%;
+    height: 100%;
+`;
+
+export type LottieType = 'CONNECT' | 'BLOCK' | 'MEMPOOL';
+
+type LottieAnimationProps = {
+    size?: number;
+    type: LottieType;
+    loop?: boolean;
+    shape?: Shape;
+    deviceModelInternal?: DeviceModelInternal;
+    deviceColor?: string;
+};
+
+export const LottieAnimation = ({
+    size,
+    type,
+    loop = false,
+    shape,
+    // if no Trezor available, show flagship model
```

Comment:
nit: this soulds like it could be used elsewhere. perhaps we can define what model is the flagship somewhere?

Reply:
Probably not in components package 😬 let's leave it for later please